src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Counter.scss';
+
+type State = {
+  count: number
+};
+
+export class Counter extends React.Component {
+  state:State = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state:State) => (
+      { count: state.count + 1 }));
+  };
+
+  add100 = () => {
+    this.setState((state:State) => (
+      { count: state.count + 100 }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0 && this.state.count > 0) {
```

Comment:
```suggestion
    if (this.state.count % 5 === 0) {
```
unnecessary check, there is no mentioned that if count === 0 it shouldn't add 100